server/controllers/match/placementRequests/bookingsController.test.ts

```diff
@@ -41,7 +41,7 @@ describe('BookingsController', () => {
       placementRequestService.getPlacementRequest.mockResolvedValue(placementRequestDetail)
 
       const query = {
-        bedSearchResult: encodeSpaceSearchResult(bedSearchResult),
+        bedSearchResult: encodeSpaceSearchResult(bedSearchResult as unknown as SpaceSearchResult),
```

Comment:
This looks a bit odd to me - wouldn't the `bedSearchResult` be for the old search and `SpaceSearchResult` be the new one?